Sources/PackageDescription/Target.swift

```diff
@@ -981,15 +986,25 @@ extension Target.Dependency {
     ///   - package: The name of the package.
     ///   - condition: A condition that limits the application of the target dependency. For example, only apply a
     ///       dependency for a specific platform.
-    @available(_PackageDescription, introduced: 5.3)
+    @available(_PackageDescription, introduced: 5.3, obsoleted: 999.0)
     public static func product(
         name: String,
         package: String,
         condition: TargetDependencyCondition? = nil
     ) -> Target.Dependency {
-        return .productItem(name: name, package: package, condition: condition)
+        return .productItem(name: name, moduleAliases: nil, package: package, condition: condition)
     }
-    
+
+    @available(_PackageDescription, introduced: 999.0)
+    public static func product(
+      name: String,
+      moduleAliases: [String: String]? = nil,
```

Comment:
I think the right order is: `name`, `package`, `moduleAliases`, `condition`: defaulted argument come after non-defaulted ones as a rule of thumb

Reply:
👍